docs/conceptual/cmake-packages.rst

```diff
@@ -151,7 +150,7 @@ Compiling device code in C++ language mode
   understanding's sake. It pre-dates the existence of HIP language support in
   CMake. If source code has HIP device code in it, it is a HIP source file
   and should be compiled as such. Only resort to the method below if your
-  HIP-enabled CMake codepath can't mandate CMake version 3.21.
+  HIP-enabled CMake code path can't mandate CMake version 3.21.
```

Comment:
@LisaDelaney We should avoid contractions in technical documentation. Let's add this to our list of guidelines. Change 'can't' to cannot.